### PR TITLE
Need to account for no line-feed in front of `DAQ`

### PR DESCRIPTION
**Pull request by BillBuilt**
While creating the datanymizer rule that would emulate our barcode generator here: https://github.com/digitalmint/barcode-generator/blob/master/scripts/generate-barcode.js

I discovered that we were not accounting for not having a line-feed in front of `DAQ`. This should reduce false-positives regarding missing serial numbers.

### CODE REVIEW

File: barcode.go [hunk 1]
```diff
@@ -179,16 +180,58 @@ func parseDate(date, fieldName string) (*time.Time, error) {
 
 // extractData extracts the information associated with 'prefix' from 'data' and return the extracted string and any errors
 func extractData(data string, prefix BarcodeDataPrefix) (string, error) {
-	re := regexp.MustCompile(fmt.Sprintf(`\n%s\s*(\S+)`, prefix))
-	match := re.FindStringSubmatch(data)
-	if len(match) > 1 {
-		return strings.TrimSpace(match[1]), nil
+	zap.S().Infow("extractData", "prefix", prefix)
```
Comment (gregwebs) [inline barcode.go:183]: can we bump the golang version and switch to slog to reduce our dependencies?

Comment (BillBuilt) [reply barcode.go:183]: IDC - zap was already in-use so i used it.... :)

**Comment by BillBuilt** [comment]
Do we need a new tag for this?

**Comment by gregwebs** [comment]
I pushed a tag v0.5.0

**Pull request merged**